### PR TITLE
Rework profession skills tab

### PR DESCRIPTION
**Pull request by pedroaugustobt**
Rework all profession skills, next week I will make all other changes to the sheet.
Note: I made a separate fix in "tab-skills.hbs/tab-skills.css", some columns in Equipment and Inventory were displaced.
image

### CODE REVIEW

File: styles/system-styles.css [hunk 1]
```diff
@@ -101,84 +101,104 @@
 }
 
 .profession-notes {
```
Comment (Stexinator) [inline styles/system-styles.css:103]: Can you move everything with profession in its name to the profession-sheet?

File: styles/system-styles.css [hunk 2]
```diff
@@ -101,84 +101,104 @@
...
-  background-color: grey;
+  background-color: #818181;
   color: white;
   display: flex;
   align-items: center;
+  border-radius: 5px 5px 0 0;
+  padding: 5px 5px 4px;
 }
 
 .defining-skill {
```
Comment (Stexinator) [inline styles/system-styles.css:140]: defining too, its also belongs to the profession

File: templates/sheets/profession-sheet.hbs [hunk 3]
```diff
@@ -39,7 +45,7 @@
                                 {{/each}}
                             {{/select}}
                         </select>
-                        <label>{{localize "WITCHER.Actor.Profession.Level"}}</label>
+                        {{!-- <label>{{localize "WITCHER.Actor.Profession.Level"}}</label> --}}
```
Comment (Stexinator) [inline templates/sheets/profession-sheet.hbs:48]: Why is this commented? If it is not needed anymore you can delete it. Git will take care of the history if it is needed again

File: styles/system-styles.css (hunk 1 above)
Comment (pedroaugustobt) [reply styles/system-styles.css:103]: Yeah, shure
I do this and remove comments

**Pull request merged**